qutebrowser/mainwindow/tabwidget.py

```diff
@@ -68,15 +67,15 @@ def __init__(self, win_id, parent=None):
         self.setStyle(TabBarStyle())
         self.setTabBar(bar)
         bar.tabCloseRequested.connect(self.tabCloseRequested)
-        bar.tabMoved.connect(functools.partial(
```

Comment:
In order to remove the signal more easily in the subclass, I removed the QTimer; Is that a good idea? Why was it there in the first place, since it doesn't look there's any difference in performance?